_docs/reference/contribute/writing-a-new-topic.md

```diff
@@ -228,15 +228,14 @@ display a config file or a test file. To do so, you can't use normal markup and
 use direct HTML. For example:
 
 ```
-{% raw %}<pre data-src="{{home}}/repos/istio/BUILD"></pre>{% endraw %}
+{% raw %}<pre data-src="https://raw.githubusercontent.com/istio/istio/master/BUILD"></pre>{% endraw %}
 ```
 
 which produces the following result:
 
-<pre data-src="{{home}}/repos/istio/BUILD"></pre>
+<pre data-src="https://raw.githubusercontent.com/istio/istio/master/BUILD"></pre>
 
-The `data-src` attribute specifies the path to the file to display. This has to be a file within the
```

Comment:
CORS is enabled on `raw.githubusercontent.com` so we don't have to do anything special here.
So long as we are linking files from github, this works ok.